### ISSUE TITLE
Support for stagger animations?

### ISSUE THREAD

**Issue by GFoley83** [minipai/ng-trans.css#6]
Not sure if this library still supported but stagger animations would be nice.

**Comment by minipai** [comment]
Do yo mean this? http://www.yearofmoo.com/2013/12/staggering-animations-in-angularjs.html

**Comment by GFoley83** [comment]
Yes just like that but triggered using classes. Exactly like how nzAnimate (https://github.com/nozzle/nzAnimate) does it (when it worked). I'd imagine the css classes could be something like `ng-trans ng-trans-stagger ng-trans-fade-right`.

**Comment by minipai** [comment]
Seems it needs a list of stagger class to define delay time. 
```
.ng-trans-stagger-100
.ng-trans-stagger-200
...
```

Sorry it's not a simple patch and I don't use Angular now, so won't add this.
But thanks for let me know nzAnimate, I'll put a link to it in README
